app/src/models.py

```diff
@@ -42,6 +42,8 @@ class User(UserMixin, db.Model):
     registration_date = db.Column(db.DateTime, default=datetime.now(timezone(timedelta(hours=-5))))
     favorite_subject = db.Column(db.String(50))
     failed_signin_attempts = db.Column(db.Integer, default=0)
+    points = db.Column(db.Integer, default=0)
```

Comment:
As previously discussed in the meeting Clara’s model can be used for this, so incorporate it if you have availability